charts/ark-solr/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for solr-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullSecrets: []
+
+#Common Parameters
+nameOverride: ""
+
+fullnameOverride: ""
+
+#Comman Labels to add to all solr resources
+commonLabels: {}
+port: 8983
+## deployment options
+# general: Minikube 
+# eks: Amazon Elastic Kubernetes Service
+deployment: ""
```

Comment:
@naveen-armedia It looks like this variable is not used anymore. Could you please remove it?

Reply:
@fabricetriboix   I thought you have used `deployment` key for `dev-deployment.yaml` file for deployment.  changed it to `platform` variable as you we are following in deploy script.

Reply:
@naveen-armedia This `platform` variable is specific to the deployment script, it is not meant to be used outside of it. Additionally, we might go to CDK for Kubernetes. And at the end of day, if you don't use a variable in the Helm chart, you should remove that variable. Keep things simple, clear and obvious.